config/constants.go

```diff
@@ -16,11 +16,33 @@ const (
 // Because Go does not support constant Structs :(
 func defaultConfig() Config {
 	return Config{
-		HistoryFilePath: defaultHistoryFile,
-		MaxHistory:      defaultMaxHist,
-		AllowDuplicates: defaultAllowDuplicates,
-		TempDirPath:     defaultTempDir,
-		LogFilePath:     defaultLogFile,
-		ThemeFilePath:   defaultThemeFile,
+		AllowDuplicates: true,
+		HistoryFilePath: "clipboard_history.json",
```

Comment:
I would prefer to keep the values used in `defaultConfig` to the constant values that are defined above. Perhaps `defaultKeyBindings` could be added as a new const instead?